addons/l10n_ca/data/tax_report.xml

```diff
@@ -0,0 +1,848 @@
+                <field name="expression_ids">
+                    <record id="l10n_ca_tr_qst_103_tag" model="account.report.expression">
+                        <field name="label">gsthst</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">CaQc103</field>
+                    </record>
+                    <record id="l10n_ca_tr_qst_203_tag" model="account.report.expression">
+                        <field name="label">qst</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">CaQc203</field>
+                    </record>
+                </field>
+            </record>
+            <record id="l10n_ca_tr_qst_104_204" model="account.report.line">
+                <field name="name">104 - 204 - Tax amounts adjustments</field>
+                <field name="code">CaQc104204</field>
+                <field name="hierarchy_level">5</field>
+                <field name="expression_ids">
+                    <record id="l10n_ca_tr_qst_104_tag" model="account.report.expression">
+                        <field name="label">gsthst</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">CaQc104</field>
+                    </record>
+                    <record id="l10n_ca_tr_qst_204_tag" model="account.report.expression">
+                        <field name="label">qst</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">CaQc204</field>
+                    </record>
+                </field>
+            </record>
+            <record id="l10n_ca_tr_qst_105_205" model="account.report.line">
+                <field name="name">105 - 205 - Tax amounts collectible and adjustments</field>
+                <field name="code">CaQc105205</field>
+                <field name="hierarchy_level">4</field>
+                <field name="expression_ids">
+                    <record id="l10n_ca_tr_qst_105_aggr" model="account.report.expression">
+                        <field name="label">gsthst</field>
+                        <field name="engine">aggregation</field>
+                        <field name="formula">CaQc103203.gsthst + CaQc104204.gsthst</field>
+                    </record>
+                    <record id="l10n_ca_tr_qst_205_aggr" model="account.report.expression">
+                        <field name="label">qst</field>
+                        <field name="engine">aggregation</field>
+                        <field name="formula">CaQc103203.qst + CaQc104204.qst</field>
+                    </record>
+                </field>
+            </record>
+            <record id="l10n_ca_tr_qst_106_206" model="account.report.line">
+                <field name="name">106 - 206 - ITCs/ITRs</field>
+                <field name="code">CaQc106206</field>
```

Comment:
Such names are wtf :face_with_spiral_eyes:

Reply:
They make sense, but I admit it's an acquired taste :p